modules/ROOT/pages/amazon/amazon-ec2-connector.adoc

```diff
@@ -311,7 +458,7 @@ Listed below are the few common use cases for the connector:
 |===
 
 [[example-use-case]]
-=== Demo Mule Application Using the Connector
+=== Demo Mule App Using the Connector
 
 This demo creates an EBS volume that can be attached to an EC2 instance in the same Availability Zone.
 
```

Comment:
Lines 467-468
. Create a new Mule project in Anypoint Studio.
. To create a separate credentials file, add the following properties to `mule-artifact.properties` and place it in the project's `src/main/resources` directory.